examples/minioinstance-kes.yaml

```diff
@@ -0,0 +1,174 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-creds-secret
+type: Opaque
+data:
+  accesskey: bWluaW8= # base 64 encoded "minio" (echo -n 'minio' | base64)
+  secretkey: bWluaW8xMjM= # based 64 encoded "minio123" (echo -n 'minio123' | base64)
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-mcs-secret
+type: Opaque
+data:
+  mcshmacjwt: WU9VUkpXVFNJR05JTkdTRUNSRVQ= # base 64 encoded "YOURJWTSIGNINGSECRET" (echo -n 'YOURJWTSIGNINGSECRET' | base64)
+  mcspbkdfpassphrase: U0VDUkVU # base 64 encoded "SECRET" (echo -n 'SECRET' | base64)
+  mcspbkdfsalt: U0VDUkVU # base 64 encoded "SECRET" (echo -n 'SECRET' | base64)
+  mcssecretkey: WU9VUk1DU1NFQ1JFVA== # base 64 encoded "YOURMCSSECRET" (echo -n 'YOURMCSSECRET' | base64)
```

Comment:
```suggestion
  MCS_HMAC_JWT_SECRET: WU9VUkpXVFNJR05JTkdTRUNSRVQ= # base 64 encoded "YOURJWTSIGNINGSECRET" (echo -n 'YOURJWTSIGNINGSECRET' | base64)
  MCS_PBKDF_PASSPHRASE: U0VDUkVU # base 64 encoded "SECRET" (echo -n 'SECRET' | base64)
  MCS_PBKDF_SALT: U0VDUkVU # base 64 encoded "SECRET" (echo -n 'SECRET' | base64)
  MCS_ACCESS_KEY: WU9VUk1DU0FDQ0VTUw== # base 64 encoded "YOURMCSACCESS" (echo -n 'YOURMCSACCESS' | base64)
  MCS_SECRET_KEY: WU9VUk1DU1NFQ1JFVA== # base 64 encoded "YOURMCSSECRET" (echo -n 'YOURMCSSECRET' | base64)
```
if we rename the secrets to use the same name MCS is expectinng for environment variables then we can just load all of them in 1 line
```
apiVersion: v1
kind: Pod
metadata:
  name: mcs
spec:
  containers:
    - name: mcs
      ...
      envFrom:
      - secretRef:
          name: minio-mcs-secret
```

Otherwise we need to map the secrets to env variables, ie:

```
    env:
      - name: MCS_HMAC_JWT_SECRET
        valueFrom:
          secretKeyRef:
            name: minio-mcs-secret
            key: mcshmacjwt
      - name: MCS_PBKDF_PASSPHRASE
        valueFrom:
          secretKeyRef:
            name: minio-mcs-secret
            key: mcspbkdfpassphrase
   ...
   ..
```

Reply:
This is a great idea, will help remove some code from Operator. But there is already a lot going on in this PR. IMO we can do this in a separate PR. I will send that soon

Reply:
sounds good nitish

Reply:
Updated in #107